index.html

```diff
@@ -4,7 +4,191 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Pricing - JS Challenge</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap"
+      rel="stylesheet"
+    />
     <link rel="stylesheet" href="css/output.css" />
   </head>
-  <body></body>
+  <body
+    class="lg:h-screen w-screen flex justify-center lg:items-center bg-[url('/assets/images/Image.jpg')] bg-no-repeat bg-cover bg-center backdrop-brightness-[0.35] relative"
+  >
+    <button
+      class="absolute top-10 right-10 hover:brightness-125 transition-all duration-300"
+    >
+      <img src="/assets/icons/Pricing Button.svg" />
+    </button>
+    <section class="flex flex-col items-center gap-20 w-full">
+      <div class="flex flex-col items-center gap-6">
+        <div class="flex flex-col text-center gap-8">
+          <h1 class="text-primary font-ubuntu font-medium text-4xl">
+            Pick the best plan for you
+          </h1>
+          <p
+            class="text-primary font-manrope font-normal text-xl tracking-wider max-w-xl leading-8"
+          >
+            You have Free Unlimited Updates and Premium Support on each package.
+          </p>
+        </div>
+
+        <div class="flex">
+          <button
+            class="font-manrope bg-primary font-bold text-dark_grey text-sm tracking-wider border border-primary rounded-l-full p-3"
+          >
+            Month View
+          </button>
+          <button
+            class="font-manrope text-sm font-medium border tracking-wider text-primary border-primary rounded-r-full p-3 bg-dark"
+          >
+            Week View
+          </button>
+        </div>
+      </div>
+      <div class="flex flex-col lg:flex-row gap-9 justify-center">
+        <div
+          class="flex flex-col items-center border border-primary py-8 px-2.5 gap-10"
+        >
+          <div class="flex flex-col items-center">
+            <h3
+              class="text-primary font-manrope font-bold text-lg tracking-wider"
```

Comment:
i see font-manrope defined many times. Try to define font-manrope to the <body> tag, so that the font is now inherited by all the child elements unless a more specific class is applied, for example when you use font-ubuntu